modules/aws-vpc.adoc

```diff
@@ -11,7 +11,7 @@ A Virtual Private Cloud (VPC) peering connection is a networking connection betw
 
 [WARNING]
 ====
-Private clusters cannot be fully deleted by {cluster-manager-first} if the VPC the cluster is installed in is peered.
+Before attempting to uninstall a cluster, you must remove any VPC peering connections from the cluster's VPC.
```

Comment:
Consider maybe (but it's your call): 

"Before you attempt to uninstall a cluster, you must remove any VPC peering connections from the cluster's VPC."

Reply:
You could offer the user a little more context for why they need to take this action. https://redhat-documentation.github.io/supplementary-style-guide/#admonitions:~:text=disregard%20this%20information.-,WARNING,-Information%20about%20potential